### ISSUE TITLE
Maybe look into autoharp?

### ISSUE THREAD

**Issue by eddelbuettel** [stat447/plr#1]
New CRAN package as of today: https://cran.r-project.org/package=autoharp

> #### autoharp: Semi-Automatic Grading of R and Rmd Scripts
>
> A customisable set of tools for assessing and grading R or R-markdown scripts from students. It allows for checking correctness of code output, runtime statistics and static code analysis. The latter feature is made possible by representing R expressions using a tree structure.

**Comment by barbehenna** [comment]
Sounds useful!

**Comment by eddelbuettel** [comment]
Yup. Some friends / aquaintances are better at walking up and down the parse tree (a wee bit esoteric for me...) but I think we wanted related features before....
